public/app/screens/main/devices/device-list/device-list.jsx

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var React = require("react");
+var Device = require('./device');
+var Room = require('./room');
+
+var DeviceList = React.createClass({
```

Comment:
I would like to reconsider the indentation rules: 4 spaces for a tab are too much.